app/controllers/appointment_summaries_controller.rb

```diff
@@ -15,11 +15,13 @@ def create
   def show
     appointment_summary = AppointmentSummary.find(params[:id])
     output_document = OutputDocument.new(appointment_summary)
+    output_document_pdf = output_document.pdf
+    OutputDocumentMailer.issue(appointment_summary, output_document_pdf).deliver_later
```

Comment:
The email address is optional. We need to put a guard in place to make sure the mailer is only triggered if we have an email address to send to.